// @mt/types-kayn/dist/asset.js
export var Asset;
(function (Asset) {
    let Ext;
    (function (Ext) {
        /** 资产类型 */
        let Type;
        (function (Type) {
            /** 单文件 */
            Type[Type["FILE"] = 0] = "FILE";
            /** 文件夹 */
            Type[Type["FOLDER"] = 1] = "FOLDER";
            /** 多版本文件 */
            Type[Type["STACK"] = 2] = "STACK";
            /** 当前资产是否是处于根目录 */
            Type[Type["ROOT"] = 3] = "ROOT";
            /** 工程文件 */
            Type[Type["ENGINEERING"] = 4] = "ENGINEERING";
            /**
             * @desc WPS文件
             * - word
             * - ppt
             * - excel
             */
            Type[Type["WPS"] = 5] = "WPS";
        })(Type = Ext.Type || (Ext.Type = {}));
        /** 媒体类型 */
        let Category;
        (function (Category) {
            /** 其它 */
            Category[Category["OTHER"] = 0] = "OTHER";
            /** 视频 */
            Category[Category["VIDEO"] = 1] = "VIDEO";
            /** 音频 */
            Category[Category["AUDIO"] = 2] = "AUDIO";
            /** 字幕 */
            Category[Category["SUBTITLE"] = 3] = "SUBTITLE";
            /** 图片 */
            Category[Category["IMAGE"] = 4] = "IMAGE";
            /**
             * @desc WPS文件
             * - word
             * - ppt
             * - excel
             */
            Category[Category["WPS"] = 5] = "WPS";
            /** .url文件 */
            Category[Category["URL"] = 6] = "URL";
        })(Category = Ext.Category || (Ext.Category = {}));
        /** 审阅状态 */
        let Label;
        (function (Label) {
            /** 重置状态 */
            Label["none"] = "none";
            /** 待审阅 */
            Label["PENDING"] = "PENDING";
            /** 审阅中 */
            Label["PROCESSING"] = "PROCESSING";
            /** 意见汇总完毕 */
            Label["CHECKEND"] = "CHECKEND";
            /** 通过 */
            Label["APPROVED"] = "APPROVED";
        })(Label = Ext.Label || (Ext.Label = {}));
        let Extra;
        (function (Extra) {
            Extra["DeleteAttach"] = "deleteAttach";
            Extra["DeleteUnusedAttach"] = "deleteUnusedAttach";
        })(Extra = Ext.Extra || (Ext.Extra = {}));
        /** 资产批量动作 */
        let AssetAction;
        (function (AssetAction) {
            /** 复制 */
            AssetAction[AssetAction["COPY"] = 1] = "COPY";
            /** 移动 */
            AssetAction[AssetAction["MOVE"] = 2] = "MOVE";
            /** 删除 */
            AssetAction[AssetAction["DEL"] = 3] = "DEL";
            /** 恢复 */
            AssetAction[AssetAction["RESTORE"] = 4] = "RESTORE";
            /** 永久删除 */
            AssetAction[AssetAction["DES"] = 5] = "DES";
            /** 转存 */
            AssetAction[AssetAction["TRANS"] = 6] = "TRANS";
            /** 导出 */
            AssetAction[AssetAction["DownloadComments"] = 7] = "DownloadComments";
            /** 设置审阅状态 */
            AssetAction[AssetAction["REVIEW"] = 8] = "REVIEW";
            /** 审批 */
            AssetAction[AssetAction["APPROVAL"] = 9] = "APPROVAL";
            /** 流程 */
            AssetAction[AssetAction["FLOW"] = 10] = "FLOW";
            /** 流程相关删除 */
            AssetAction[AssetAction["FlowDel"] = 11] = "FlowDel";
        })(AssetAction = Ext.AssetAction || (Ext.AssetAction = {}));
    })(Ext = Asset.Ext || (Asset.Ext = {}));
})(Asset || (Asset = {}));
