// @mt/types-kayn/dist/comment.js
export var Comment;
(function (Comment) {
    let Ext;
    (function (Ext) {
        let Level;
        (function (Level) {
            Level[Level["ROOT"] = 0] = "ROOT";
            Level[Level["COMMENT"] = 1] = "COMMENT";
            Level[Level["REPLIES"] = 2] = "REPLIES";
        })(Level = Ext.Level || (Ext.Level = {}));
    })(Ext = Comment.Ext || (Comment.Ext = {}));
    let Param;
    (function (Param) {
        let ReplyFilter;
        (function (ReplyFilter) {
            /** 全部批注 */
            ReplyFilter["all"] = "all";
            /** 已修改批注 */
            ReplyFilter["completed"] = "completed";
            /** 未修改批注 */
            ReplyFilter["uncompleted"] = "uncompleted";
            /** 全员可见批注 */
            ReplyFilter["all_member"] = "all_member";
            /** 项目成员可见批注 */
            ReplyFilter["project_member"] = "project_member";
        })(ReplyFilter = Param.ReplyFilter || (Param.ReplyFilter = {}));
    })(Param = Comment.Param || (Comment.Param = {}));
    let CommentState;
    (function (CommentState) {
        /**
         * 0 正常
         */
        CommentState["Normal"] = "0";
        /**
         * 1 已编辑
         */
        CommentState["Edited"] = "1";
        /**
         * 2 已删除
         */
        CommentState["Deleted"] = "2";
    })(CommentState = Comment.CommentState || (Comment.CommentState = {}));
})(Comment || (Comment = {}));
