// @mt/types-kayn/dist/filter.js
export var Filter;
(function (Filter) {
    let Ext;
    (function (Ext) {
        let ObjKey;
        (function (ObjKey) {
            /**
             * 父文件夹ID
             * ***
             * 当搜索某一个文件夹下的资产时，需要传入该文件夹的folder_id用来搜索版本内资产
             */
            ObjKey["PARENT_ID"] = "parent_id";
            /**
             * 标签
             */
            ObjKey["TAGS"] = "tags";
            /**
             * 文件宽\
             * file.info.Streams.VideoStreamList.VideoStream.Width
             */
            ObjKey["WIDTH"] = "width";
            /**
             * 文件高
             * ***
             * file.info.Streams.VideoStreamList.VideoStream.Height
             */
            ObjKey["HEIGHT"] = "height";
            /**
             * 文件持续时长
             * ***
             * file.duration
             */
            ObjKey["DURATION"] = "duration";
            /**
             * 文件名称
             * ***
             * title （ObjKey里面的title不是给全局搜索用的， 而是在智能筛选用）
             */
            ObjKey["TITLE"] = "title";
            /**
             * ~~资产类型~~
             */
            ObjKey["TYPE"] = "type";
            /**
             * 文件类型
             * ***
             * {@link Asset.Ext.Category}
             */
            ObjKey["CATEGORY"] = "category";
            /**
             * 文件大小
             */
            ObjKey["SIZE"] = "size";
            /**
             * 创建时间
             */
            ObjKey["CREATED_AT"] = "created_at";
            /**
             * 更新时间
             */
            ObjKey["UPDATED_AT"] = "updated_at";
            /**
             * 文件描述
             */
            ObjKey["DESCRIPTION"] = "description";
            /**
             * 资源库根ID
             * - 个人资源库 -> space_id
             * - 团队资源库 -> disk_org_space
             */
            ObjKey["SPACE_ID"] = "space_id";
            /**
             * 项目id
             */
            ObjKey["PROJECT_ID"] = "project_id";
            /**
            * 文件夹路径
            */
            ObjKey["FULL_PATH"] = "full_path";
            /**
             * btn,是线上前端bug导致的脏数据
             */
            ObjKey["BTN"] = "btn";
        })(ObjKey = Ext.ObjKey || (Ext.ObjKey = {}));
        let ObjValueOfKey;
        (function (ObjValueOfKey) {
            ObjValueOfKey["lt"] = "lt";
            ObjValueOfKey["lte"] = "lte";
            ObjValueOfKey["eq"] = "eq";
            ObjValueOfKey["neq"] = "neq";
            ObjValueOfKey["gte"] = "gte";
            ObjValueOfKey["gt"] = "gt";
            ObjValueOfKey["like"] = "like";
            ObjValueOfKey["nlike"] = "nlike";
            ObjValueOfKey["unit"] = "unit";
            ObjValueOfKey["properties"] = "properties";
        })(ObjValueOfKey = Ext.ObjValueOfKey || (Ext.ObjValueOfKey = {}));
    })(Ext = Filter.Ext || (Filter.Ext = {}));
})(Filter || (Filter = {}));
