// @mt/types-kayn/dist/flow.js
export var Flow;
(function (Flow) {
    let Ext;
    (function (Ext) {
        let NodeGroup;
        (function (NodeGroup) {
            NodeGroup[NodeGroup["INIT"] = 1] = "INIT";
            NodeGroup[NodeGroup["UNDERWAY"] = 2] = "UNDERWAY";
            NodeGroup[NodeGroup["COMPLETE"] = 3] = "COMPLETE";
            NodeGroup[NodeGroup["END"] = 4] = "END";
        })(NodeGroup = Ext.NodeGroup || (Ext.NodeGroup = {}));
        /** 任务类型 */
        let NodeTaskType;
        (function (NodeTaskType) {
            /** 无意义 */
            NodeTaskType[NodeTaskType["DEFAULT"] = 0] = "DEFAULT";
            /** 待办任务 */
            NodeTaskType[NodeTaskType["TODO"] = 1] = "TODO";
            /** 审批任务 */
            NodeTaskType[NodeTaskType["APPROVAL"] = 2] = "APPROVAL";
        })(NodeTaskType = Ext.NodeTaskType || (Ext.NodeTaskType = {}));
        /** 审批类型 */
        let AuditType;
        (function (AuditType) {
            /** 无意义 */
            AuditType[AuditType["DEFAULT"] = 0] = "DEFAULT";
            /** 无需审核 */
            AuditType[AuditType["NONE"] = 1] = "NONE";
            /** 或签 */
            AuditType[AuditType["ORSIGN"] = 2] = "ORSIGN";
            /** 会签 */
            AuditType[AuditType["COUNTERSIGN"] = 3] = "COUNTERSIGN";
        })(AuditType = Ext.AuditType || (Ext.AuditType = {}));
        /** 处理结果 */
        // 处理结果（0无意义、1未处理 2已阅、3通过、4驳回）
        let Result;
        (function (Result) {
            /** 无意义 */
            Result[Result["DEFAULT"] = 0] = "DEFAULT";
            /** 未处理 */
            Result[Result["UNREAD"] = 1] = "UNREAD";
            /** 已阅 */
            Result[Result["READ"] = 2] = "READ";
            /** 通过 */
            Result[Result["RESOLVE"] = 3] = "RESOLVE";
            /** 驳回 */
            Result[Result["REJECT"] = 4] = "REJECT";
        })(Result = Ext.Result || (Ext.Result = {}));
        /** 处理状态 */
        let State;
        (function (State) {
            /** 无意义 */
            State[State["DEFAULT"] = 0] = "DEFAULT";
            /** 未处理 */
            State[State["UNDO"] = 1] = "UNDO";
            /** 处理中 */
            State[State["DOING"] = 2] = "DOING";
            /** 已处理 */
            State[State["DONE"] = 3] = "DONE";
            /** 无需处理 */
            State[State["NONEED"] = 4] = "NONEED";
            /** 撤销 */
            State[State["CANCEL"] = 5] = "CANCEL";
            /** 销毁 */
            State[State["DESTROY"] = 6] = "DESTROY";
        })(State = Ext.State || (Ext.State = {}));
    })(Ext = Flow.Ext || (Flow.Ext = {}));
})(Flow || (Flow = {}));
