// @mt/types-kayn/dist/guide.js
/** 新手引导 */
export var Guide;
(function (Guide) {
    /** 引导状态 */
    let GuideStatus;
    (function (GuideStatus) {
        /**  默认值 */
        GuideStatus["None"] = "0";
        /** 新用户 */
        GuideStatus["New"] = "1";
        /** 老用户 */
        GuideStatus["Old"] = "2";
        /** 重置状态 */
        GuideStatus["Reset"] = "3";
        /** 完成引导 */
        GuideStatus["Finish"] = "4";
    })(GuideStatus = Guide.GuideStatus || (Guide.GuideStatus = {}));
    /** 引导端 */
    let GuideClient;
    (function (GuideClient) {
        /** web端 */
        GuideClient["Sona"] = "sona";
        /** 客户端 */
        GuideClient["Ekko"] = "ekko";
        /** 小程序 */
        GuideClient["Neeko"] = "neeko";
    })(GuideClient = Guide.GuideClient || (Guide.GuideClient = {}));
    /**
     * 引导类型细分
     * TODO: 历史原因
     * 客户端&web端 Global 和 Welcome 表示的意思互换了
     */
    let Type;
    (function (Type) {
        /** web端 */
        let SonaGuideType;
        (function (SonaGuideType) {
            /** 用户触达 */
            SonaGuideType["CeoLetter"] = "ceo_letter";
            /** 欢迎页 */
            SonaGuideType["Global"] = "global";
            /** 全局 */
            SonaGuideType["Welcome"] = "welcome";
            /** 项目 */
            SonaGuideType["Project"] = "project";
            /** 资源库 */
            SonaGuideType["Resource"] = "resource";
            /** 分享 */
            SonaGuideType["Share"] = "share";
            /** 批注 */
            SonaGuideType["Comment"] = "comment";
            /** 审阅 */
            SonaGuideType["Review"] = "review";
            /** 通讯录 */
            SonaGuideType["AddressBook"] = "address_book";
            /** 菜单缩进 */
            SonaGuideType["Interaction"] = "interaction";
            /** 团队试用引导 */
            SonaGuideType["TrialNotification"] = "trial_notification";
        })(SonaGuideType = Type.SonaGuideType || (Type.SonaGuideType = {}));
        /** 客户端 */
        let EkkoGuideType;
        (function (EkkoGuideType) {
            /** 用户触达 */
            EkkoGuideType["CeoLetter"] = "ceo_letter";
        })(EkkoGuideType = Type.EkkoGuideType || (Type.EkkoGuideType = {}));
        /** 小程序端 */
        let NeekoGuideType;
        (function (NeekoGuideType) {
            /** 欢迎页 */
            NeekoGuideType["Welcome"] = "welcome";
            /** 首页引导 */
            NeekoGuideType["Home"] = "home";
            /** 审阅 */
            NeekoGuideType["Review"] = "review";
            /** 视频操作提示 */
            NeekoGuideType["Video"] = "video";
            /** 用户触达 */
            NeekoGuideType["CeoLetter"] = "ceo_letter";
        })(NeekoGuideType = Type.NeekoGuideType || (Type.NeekoGuideType = {}));
    })(Type = Guide.Type || (Guide.Type = {}));
})(Guide || (Guide = {}));
