// @mt/types-kayn/dist/limit.js
/** 限制模块 */
export var Limit;
(function (Limit) {
    /** 限制模块 */
    let LimitModule;
    (function (LimitModule) {
        /** 资产相关 */
        LimitModule["Asset"] = "asset";
        /** 批注相关 */
        LimitModule["Comment"] = "comment";
        /** 媒体相关 */
        LimitModule["Media"] = "media";
        /** 分享先关 */
        LimitModule["Review"] = "review";
        /** 团队人员限制 */
        LimitModule["OrgMembers"] = "org_members";
        /** 组织权限 */
        LimitModule["Organization"] = "organization";
        /** 项目权限 */
        LimitModule["Project"] = "project";
        /** 项目人员限制 */
        LimitModule["ProjectMembers"] = "project_members";
        /** 资源库权益 */
        LimitModule["Resource"] = "resource";
        /** 容量限制 */
        LimitModule["Storage"] = "storage";
        /** 服务支持 */
        LimitModule["SupportedService"] = "supported_service";
        /** 链接 */
        LimitModule["Link"] = "link";
        /** 流程 */
        LimitModule["Flow"] = "flow";
        /** 扩展 */
        LimitModule["App"] = "app";
    })(LimitModule = Limit.LimitModule || (Limit.LimitModule = {}));
    /** 服务支持类型 */
    let SupportedServiceType;
    (function (SupportedServiceType) {
        /** 扩容服务 */
        SupportedServiceType["EXPANSION"] = "expansion";
        /** VIP客户服务 */
        SupportedServiceType["VIP_CS"] = "vip_cs";
    })(SupportedServiceType = Limit.SupportedServiceType || (Limit.SupportedServiceType = {}));
})(Limit || (Limit = {}));
