// @mt/types-kayn/dist/link.js
export var Link;
(function (Link) {
    let Ext;
    (function (Ext) {
        /**
         * 链接类型
         * 0:未知
         * 1:分享
         * 2:审阅
         * 3:演示
         * 4:收集
         * 5:交付
         */
        let Type;
        (function (Type) {
            Type[Type["Unknown"] = 0] = "Unknown";
            Type[Type["Share"] = 1] = "Share";
            Type[Type["Review"] = 2] = "Review";
            Type[Type["Demo"] = 3] = "Demo";
            Type[Type["Collect"] = 4] = "Collect";
            Type[Type["Delivery"] = 5] = "Delivery";
        })(Type = Ext.Type || (Ext.Type = {}));
        /**
         * 链接状态
         * 0:未知
         * 1:启用
         * 2:关闭
         */
        let State;
        (function (State) {
            State[State["Unknown"] = 0] = "Unknown";
            State[State["Enable"] = 1] = "Enable";
            State[State["Disable"] = 2] = "Disable";
        })(State = Ext.State || (Ext.State = {}));
        /**
         * 链接来源
         * 0:未知
         * 1:团队公共资源库
         * 2:团队个人资源库
         * 3:项目
         */
        let SourceType;
        (function (SourceType) {
            SourceType[SourceType["Unknown"] = 0] = "Unknown";
            SourceType[SourceType["PublicDisk"] = 1] = "PublicDisk";
            SourceType[SourceType["PersonalDisk"] = 2] = "PersonalDisk";
            SourceType[SourceType["Project"] = 3] = "Project";
        })(SourceType = Ext.SourceType || (Ext.SourceType = {}));
        /**
         * 批注类型
         */
        let ReviewCommentType;
        (function (ReviewCommentType) {
            /** 文件详情/审阅 */
            ReviewCommentType[ReviewCommentType["Review"] = 1] = "Review";
            /** 演示 */
            ReviewCommentType[ReviewCommentType["Demo"] = 2] = "Demo";
        })(ReviewCommentType = Ext.ReviewCommentType || (Ext.ReviewCommentType = {}));
    })(Ext = Link.Ext || (Link.Ext = {}));
})(Link || (Link = {}));
