// @mt/types-kayn/dist/log.js
export var Log;
(function (Log) {
    let Ext;
    (function (Ext) {
        /** 审阅状态 */
        let MarkType;
        (function (MarkType) {
            /** 重置 */
            MarkType[MarkType["DEFAULT"] = 0] = "DEFAULT";
            /** 待审阅 */
            MarkType[MarkType["PENDING"] = 1] = "PENDING";
            /** 审阅中 */
            MarkType[MarkType["PROCESSING"] = 2] = "PROCESSING";
            /** 意见汇总完毕 */
            MarkType[MarkType["CHECKEND"] = 3] = "CHECKEND";
            /** 通过 */
            MarkType[MarkType["APPROVED"] = 4] = "APPROVED";
            /** 自定义 */
            MarkType[MarkType["CUSTOMIZE_1"] = 5] = "CUSTOMIZE_1";
            /** 自定义 */
            MarkType[MarkType["CUSTOMIZE_2"] = 6] = "CUSTOMIZE_2";
        })(MarkType = Ext.MarkType || (Ext.MarkType = {}));
        /** 自定义审阅状态类型 */
        let CustomMarkType;
        (function (CustomMarkType) {
            /** 开启 */
            CustomMarkType[CustomMarkType["Open"] = 0] = "Open";
            /** 关闭 */
            CustomMarkType[CustomMarkType["Close"] = 1] = "Close";
        })(CustomMarkType = Ext.CustomMarkType || (Ext.CustomMarkType = {}));
        let OrgMemberRole;
        (function (OrgMemberRole) {
            /** 成员 */
            OrgMemberRole[OrgMemberRole["MEMBER"] = 1] = "MEMBER";
            /** 管理员 */
            OrgMemberRole[OrgMemberRole["ADMIN"] = 2] = "ADMIN";
            /** 创建者 */
            OrgMemberRole[OrgMemberRole["CREATOR"] = 3] = "CREATOR";
        })(OrgMemberRole || (OrgMemberRole = {}));
        /** 下载类型 */
        let DownloadType;
        (function (DownloadType) {
            /** 未知文件 */
            DownloadType[DownloadType["UNKNOW"] = 0] = "UNKNOW";
            /** 资产 */
            DownloadType[DownloadType["ASSET"] = 1] = "ASSET";
            /** 缩略图 */
            DownloadType[DownloadType["THUMBNALS"] = 2] = "THUMBNALS";
            /** 代理文件 */
            DownloadType[DownloadType["PROXYFILE"] = 3] = "PROXYFILE";
        })(DownloadType = Ext.DownloadType || (Ext.DownloadType = {}));
        let LogOrgInvite;
        (function (LogOrgInvite) {
            /** 无需审批 */
            LogOrgInvite["Invite"] = "invite";
            /** 通过审批 */
            LogOrgInvite["Approval"] = "approval";
        })(LogOrgInvite = Ext.LogOrgInvite || (Ext.LogOrgInvite = {}));
        /** 操作主体类型 */
        let LogType;
        (function (LogType) {
            /** 未知 */
            LogType[LogType["UNKNOW"] = 0] = "UNKNOW";
            /** 资产 */
            LogType[LogType["ASSET"] = 1] = "ASSET";
            /** 项目 */
            LogType[LogType["PROJECT"] = 2] = "PROJECT";
            /** 组织 */
            LogType[LogType["ORG"] = 3] = "ORG";
            /** 团队资源库 */
            LogType[LogType["TEAM_RESOURCE"] = 4] = "TEAM_RESOURCE";
            /** 个人资源库 */
            LogType[LogType["PERSONAL_RESOURCE"] = 5] = "PERSONAL_RESOURCE";
            /** 团队成员 */
            LogType[LogType["TEAM_MEMBER"] = 6] = "TEAM_MEMBER";
            /** 项目成员 */
            LogType[LogType["PROJECT_MEMBER"] = 7] = "PROJECT_MEMBER";
            /** 团队部门 */
            LogType[LogType["TEAM_DEPARTMENT"] = 8] = "TEAM_DEPARTMENT";
            /** 项目分享 */
            LogType[LogType["PROJECT_SHARE"] = 9] = "PROJECT_SHARE";
            /** 个人资源库分享 */
            LogType[LogType["PERSONAL_RESOURCE_SHARE"] = 10] = "PERSONAL_RESOURCE_SHARE";
            /** 批注 */
            LogType[LogType["COMMENT"] = 11] = "COMMENT";
            /** 团队分组成员 */
            LogType[LogType["TEAM_GOURP_MEMBER"] = 12] = "TEAM_GOURP_MEMBER";
            /** 下载 */
            LogType[LogType["DOWNLOAD"] = 13] = "DOWNLOAD";
            /** 文件收集 */
            LogType[LogType["FILE_COLLECT"] = 19] = "FILE_COLLECT";
            /** 流程 */
            LogType[LogType["Flow"] = 23] = "Flow";
            /** 项目流程 */
            LogType[LogType["ProjectFlow"] = 24] = "ProjectFlow";
            /** 流程节点 */
            LogType[LogType["FlowNode"] = 25] = "FlowNode";
            /** 流程节点任务 */
            LogType[LogType["FlowNodeTask"] = 26] = "FlowNodeTask";
            /** 流程节点任务成员 */
            LogType[LogType["FlowTplNodeMember"] = 27] = "FlowTplNodeMember";
        })(LogType = Ext.LogType || (Ext.LogType = {}));
    })(Ext = Log.Ext || (Log.Ext = {}));
    /** 自定义logo类型操作 */
    let CustomLogoOpration;
    (function (CustomLogoOpration) {
        /** 自定义团队logo */
        CustomLogoOpration["ORG_LOGO"] = "orglogo.logo";
        /** 自定义团队logo开关 */
        CustomLogoOpration["ORG_SWITCH"] = "orglogo.switch";
        /** 自定义团队logo名称 */
        CustomLogoOpration["ORG_ORGNAME"] = "orglogo.orgname";
        /** 自定义分享链接logo */
        CustomLogoOpration["SHARE_LOGO"] = "sharelogo.logo";
        /** 自定义分享链接logo开关 */
        CustomLogoOpration["SHARE_SWITCH"] = "sharelogo.switch";
        /** 自定义分享链接logo名称 */
        CustomLogoOpration["SHARE_ORGNAME"] = "sharelogo.orgname";
        /** 自定义品牌展示封面 */
        CustomLogoOpration["BRAND_LOGO"] = "brandlogo.logo";
        /** 自定义品牌展示封面开关 */
        CustomLogoOpration["BRAND_SWITCH"] = "brandlogo.switch";
        /** 自定义小程序分享链接封面 */
        CustomLogoOpration["MP_SHARE_LOGO"] = "mp_sharelogo.logo";
        /** 自定义小程序分享链接封面开关 */
        CustomLogoOpration["MP_SHARE_SWITCH"] = "mp_sharelogo.switch";
        /** 自定义小程序团队邀请链接封面 */
        CustomLogoOpration["MP_ORG_INVITE_LOGO"] = "mp_orginvitelogo.logo";
        /** 自定义小程序团队邀请链接封面开关 */
        CustomLogoOpration["MP_ORG_INVITE_SWITCH"] = "mp_orginvitelogo.switch";
        /** 自定义小程序项目邀请链接封面 */
        CustomLogoOpration["MP_PROJECT_INVITE_LOGO"] = "mp_projectinvitelogo.logo";
        /** 自定义小程序项目邀请链接封面开关 */
        CustomLogoOpration["MP_PROJECT_INVITE_SWITCH"] = "mp_projectinvitelogo.switch";
    })(CustomLogoOpration = Log.CustomLogoOpration || (Log.CustomLogoOpration = {}));
})(Log || (Log = {}));
