// @mt/types-kayn/dist/messageRobot.js
/** 项目消息机器人表 */
export var MessageRobot;
(function (MessageRobot) {
    /** 事件类型 0未知类型;1向项目成员加入;2文件批注动态;3文件审阅状态变更 */
    let EventType;
    (function (EventType) {
        /** 未知类型 */
        EventType[EventType["UnknownType"] = 0] = "UnknownType";
        /** 向项目成员加入 */
        EventType[EventType["ProjectJoin"] = 1] = "ProjectJoin";
        /** 文件批注动态 */
        EventType[EventType["AssetsComments"] = 2] = "AssetsComments";
        /** 文件审阅状态变更 */
        EventType[EventType["AssetsReview"] = 3] = "AssetsReview";
    })(EventType = MessageRobot.EventType || (MessageRobot.EventType = {}));
    /** 项目名称 机器人 1开启；2关闭 */
    let MessageRobotState;
    (function (MessageRobotState) {
        /** 项目名称 机器人 开启 */
        MessageRobotState[MessageRobotState["open"] = 1] = "open";
        /** 项目名称 机器人 关闭 */
        MessageRobotState[MessageRobotState["close"] = 2] = "close";
    })(MessageRobotState = MessageRobot.MessageRobotState || (MessageRobot.MessageRobotState = {}));
})(MessageRobot || (MessageRobot = {}));
