// @mt/types-kayn/dist/message.js
export var Message;
(function (Message) {
    let Ext;
    (function (Ext) {
        /** message topic */
        let Reason;
        (function (Reason) {
            /** 资产创建 */
            Reason["event.asset.created"] = "event.asset.created";
            /** 资产删除 */
            Reason["event.asset.deleted"] = "event.asset.deleted";
            /** 版本更新 */
            Reason["event.asset.stacked"] = "event.asset.stacked";
            /** 审阅状态变更 */
            Reason["event.asset.status.changed"] = "event.asset.status.changed";
            /** 资产重命名 */
            Reason["event.asset.updated"] = "event.asset.updated";
            /** 评论状态变更 */
            Reason["event.comment.complete.changed"] = "event.comment.complete.changed";
            /** 评论添加 */
            Reason["event.comment.created"] = "event.comment.created";
            /** 评论删除 */
            Reason["event.comment.deleted"] = "event.comment.deleted";
            /** 项目申请通过 */
            Reason["event.member.accepted"] = "event.member.accepted";
            /** 项目申请加入 */
            Reason["event.member.applied"] = "event.member.applied";
            /** 项目成员移除 */
            Reason["event.member.deleted"] = "event.member.deleted";
            /** 项目邀请 */
            Reason["event.member.invited"] = "event.member.invited";
            /** 项目成员加入 */
            Reason["event.member.joined"] = "event.member.joined";
            /** 项目成员离开 */
            Reason["event.member.leaved"] = "event.member.leaved";
            /** 项目成员申请被拒绝 */
            Reason["event.member.rejected"] = "event.member.rejected";
            /** 项目成员权限变更 */
            Reason["event.member.role.changed"] = "event.member.role.changed";
            /** 项目删除 */
            Reason["event.project.deleted"] = "event.project.deleted";
            /** 项目转让 */
            Reason["event.project.transferred"] = "event.project.transferred";
            /** 申请加入团队(团队创建者、管理员 可以接收到） */
            Reason["event.organization.member.applied"] = "event.organization.member.applied";
            /** 被添加进团队（被添加者 可以接收到） */
            Reason["event.organization.member.added"] = "event.organization.member.added";
            /** 成功加入团队（团队创建者、管理员、申请人 可以接收到） */
            Reason["event.organization.member.joined"] = "event.organization.member.joined";
            /**  被移除出团队（团队创建者、管理员、被移除的人 可以接收到） */
            Reason["event.organization.member.deleted"] = "event.organization.member.deleted";
            /** 系统消息 */
            Reason["System"] = "21000001";
            /** 评论消息 */
            Reason["Comment"] = "21020002";
            /** 版本变更消息 */
            Reason["Version"] = "21020003";
        })(Reason = Ext.Reason || (Ext.Reason = {}));
        let Type;
        (function (Type) {
            Type["Accept"] = "Accept";
            Type["Add"] = "Add";
            Type["Announce"] = "Announce";
            Type["Arrive"] = "Arrive";
            Type["Block"] = "Block";
            Type["Create"] = "Create";
            Type["Delete"] = "Delete";
            Type["Dislike"] = "Dislike";
            Type["Flag"] = "Flag";
            Type["Follow"] = "Follow";
            Type["Ignore"] = "Ignore";
            Type["Invite"] = "Invite";
            Type["Join"] = "Join";
            Type["Leave"] = "Leave";
            Type["Like"] = "Like";
            Type["Listen"] = "Listen";
            Type["Move"] = "Move";
            Type["Offer"] = "Offer";
            Type["Question"] = "Question";
            Type["Reject"] = "Reject";
            Type["Read"] = "Read";
            Type["Remove"] = "Remove";
            Type["TentativeReject"] = "TentativeReject";
            Type["TentativeAccept"] = "TentativeAccept";
            Type["Travel"] = "Travel";
            Type["Undo"] = "Undo";
            Type["Update"] = "Update";
            Type["View"] = "View";
        })(Type = Ext.Type || (Ext.Type = {}));
    })(Ext = Message.Ext || (Message.Ext = {}));
})(Message || (Message = {}));
