// @mt/types-kayn/dist/org.js
import { Limit } from '.';
export var Org;
(function (Org) {
    let Ext;
    (function (Ext) {
        /**
         * 组织类型
         * - PERSONAL 个人
         * - TEAM 团队
         * - '' 无版本类型
         */
        let Plan;
        (function (Plan) {
            /** 个人版 */
            Plan["personal"] = "PERSONAL";
            /** 团队版 */
            Plan["team"] = "TEAM";
            /** 无版本类型 */
            Plan["NONE"] = "";
        })(Plan = Ext.Plan || (Ext.Plan = {}));
        /**
         * 当前组织的套餐类型
         */
        let InterestSubType;
        (function (InterestSubType) {
            /** 未知 */
            InterestSubType[InterestSubType["Rights_UnKnown"] = 0] = "Rights_UnKnown";
            /** 试用 */
            InterestSubType[InterestSubType["Rights_Trial"] = 1] = "Rights_Trial";
            /** 免费 */
            InterestSubType[InterestSubType["Rights_Free"] = 2] = "Rights_Free";
            /** A1基础版 */
            InterestSubType[InterestSubType["Rights_A1"] = 3] = "Rights_A1";
            /** A2高级版 */
            InterestSubType[InterestSubType["Rights_A2"] = 4] = "Rights_A2";
            /** B1基础版 */
            InterestSubType[InterestSubType["Rights_B1"] = 5] = "Rights_B1";
            /** B2高级版 */
            InterestSubType[InterestSubType["Rights_B2"] = 6] = "Rights_B2";
            /** C1团队旗舰版 */
            InterestSubType[InterestSubType["Rights_C1"] = 7] = "Rights_C1";
            /** 定制版本 */
            InterestSubType[InterestSubType["Rights_D"] = 8] = "Rights_D";
            // 过度版本
            /**个人版基础版 */
            InterestSubType[InterestSubType["Rights_OldPersonalStandard"] = 10001] = "Rights_OldPersonalStandard";
            /** 个人专业版 */
            InterestSubType[InterestSubType["Rights_OldPersonalPro"] = 10002] = "Rights_OldPersonalPro";
            /** 个人旗舰版 */
            InterestSubType[InterestSubType["Rights_OldPersonalUltimate"] = 10003] = "Rights_OldPersonalUltimate";
            /** 团队基础版 */
            InterestSubType[InterestSubType["Rights_OldTeamStandard"] = 10004] = "Rights_OldTeamStandard";
            /** 团队专业版 */
            InterestSubType[InterestSubType["Rights_OldTeamPro"] = 10005] = "Rights_OldTeamPro";
            /** 团队旗舰版 */
            InterestSubType[InterestSubType["Rights_OldTeamUltimate"] = 10006] = "Rights_OldTeamUltimate";
        })(InterestSubType = Ext.InterestSubType || (Ext.InterestSubType = {}));
    })(Ext = Org.Ext || (Org.Ext = {}));
    /** 自定义logo类型 */
    let CustomLogoType;
    (function (CustomLogoType) {
        /** 团队 */
        CustomLogoType["ORG"] = "orglogo";
        /** 分享链接 */
        CustomLogoType["SHARE"] = "sharelogo";
        /** 品牌展示位 */
        CustomLogoType["BRAND"] = "brandlogo";
        /** 小程序分享链接封面 */
        CustomLogoType["MP_SHARE"] = "mp_sharelogo";
        /** 小程序团队邀请链接封面 */
        CustomLogoType["MP_ORG_INVITE"] = "mp_orginvitelogo";
        /** 小程序项目邀请链接封面 */
        CustomLogoType["MP_PROJECT_INVITE"] = "mp_projectinvitelogo";
    })(CustomLogoType = Org.CustomLogoType || (Org.CustomLogoType = {}));
    let PrivacyHiddenSwitch;
    (function (PrivacyHiddenSwitch) {
        PrivacyHiddenSwitch["Open"] = "1";
        PrivacyHiddenSwitch["Close"] = "0";
    })(PrivacyHiddenSwitch = Org.PrivacyHiddenSwitch || (Org.PrivacyHiddenSwitch = {}));
})(Org || (Org = {}));
