// @mt/types-kayn/dist/pay.js
export var Pay;
(function (Pay) {
    /** 订单类型 */
    let OrderType;
    (function (OrderType) {
        /** 新开通 */
        OrderType[OrderType["NEW"] = 1] = "NEW";
        /** 续费 */
        OrderType[OrderType["RENEW"] = 2] = "RENEW";
        /** 增购 */
        OrderType[OrderType["INCREASE"] = 3] = "INCREASE";
        /** 升级 */
        OrderType[OrderType["UPGRADE"] = 4] = "UPGRADE";
    })(OrderType = Pay.OrderType || (Pay.OrderType = {}));
    /** 订单状态 */
    let OrderStatus;
    (function (OrderStatus) {
        /** 未知状态 */
        OrderStatus[OrderStatus["UNKNOW"] = 0] = "UNKNOW";
        /** 待付款 */
        OrderStatus[OrderStatus["UNPAID"] = 1] = "UNPAID";
        /** 付款中 */
        OrderStatus[OrderStatus["INPAY"] = 2] = "INPAY";
        /** 已完成 */
        OrderStatus[OrderStatus["COMPLETED"] = 3] = "COMPLETED";
        /** 待确认 */
        OrderStatus[OrderStatus["WAITCONFIRM"] = 4] = "WAITCONFIRM";
        /** 已确认 */
        OrderStatus[OrderStatus["CONFIRMED"] = 5] = "CONFIRMED";
        /** 已取消 */
        OrderStatus[OrderStatus["CANCELED"] = 6] = "CANCELED";
        /** 异常订单 */
        OrderStatus[OrderStatus["ABNORMAL"] = 7] = "ABNORMAL";
        /** 已关闭 */
        OrderStatus[OrderStatus["CLOSED"] = 8] = "CLOSED";
    })(OrderStatus = Pay.OrderStatus || (Pay.OrderStatus = {}));
    /** 支付类型 当前只支持3和7和10 */
    let PayType;
    (function (PayType) {
        /** 待付款 */
        PayType[PayType["UNPAID"] = 0] = "UNPAID";
        /** 支付宝app */
        PayType[PayType["AliAPP"] = 1] = "AliAPP";
        /** 支付宝手机网站 */
        PayType[PayType["AliH5"] = 2] = "AliH5";
        /** 支付宝pc */
        PayType[PayType["AliPC"] = 3] = "AliPC";
        /** 苹果内购 */
        PayType[PayType["Iphone"] = 4] = "Iphone";
        /** 微信app */
        PayType[PayType["WechatAPP"] = 5] = "WechatAPP";
        /** 微信手机网站 */
        PayType[PayType["WechatH5"] = 6] = "WechatH5";
        /** 微信扫码 */
        PayType[PayType["WechatQrcode"] = 7] = "WechatQrcode";
        /** 微信JSAPI */
        PayType[PayType["WechatJsApi"] = 8] = "WechatJsApi";
        /** 微信小程序 */
        PayType[PayType["WechatApplet"] = 9] = "WechatApplet";
        /** 对公转账 */
        PayType[PayType["Public"] = 10] = "Public";
    })(PayType = Pay.PayType || (Pay.PayType = {}));
    /** 扩容类型 */
    let DilatationType;
    (function (DilatationType) {
        /** 未知 */
        DilatationType["Unknow"] = "0";
        /** 空间 */
        DilatationType["Space"] = "1";
        /** 席位 */
        DilatationType["Seat"] = "2";
    })(DilatationType = Pay.DilatationType || (Pay.DilatationType = {}));
    /** 退款状态 */
    let RefundStatus;
    (function (RefundStatus) {
        /** 未退款 */
        RefundStatus[RefundStatus["Unanswered"] = 1] = "Unanswered";
        /** 部分退款 */
        RefundStatus[RefundStatus["Partial"] = 2] = "Partial";
        /** 全部退款 */
        RefundStatus[RefundStatus["All"] = 3] = "All";
    })(RefundStatus = Pay.RefundStatus || (Pay.RefundStatus = {}));
    /** 认款状态 */
    let TransferStatus;
    (function (TransferStatus) {
        /** 未知状态 */
        TransferStatus[TransferStatus["UNKNOW"] = 0] = "UNKNOW";
        /** 待支付[待提交] */
        TransferStatus[TransferStatus["UNPAID"] = 1] = "UNPAID";
        /** 支付中[待确认3] */
        TransferStatus[TransferStatus["INPAY"] = 2] = "INPAY";
        /** 支付成功[已确认款项] */
        TransferStatus[TransferStatus["COMPLETED"] = 3] = "COMPLETED";
        /** 支付失败[待重新提交] */
        TransferStatus[TransferStatus["FAILED"] = 4] = "FAILED";
        /** 已关闭[已经放弃提交] */
        TransferStatus[TransferStatus["CLOSED"] = 5] = "CLOSED";
    })(TransferStatus = Pay.TransferStatus || (Pay.TransferStatus = {}));
    /** 操作动作类型 */
    let RemitAciton;
    (function (RemitAciton) {
        /** 确认收款 */
        RemitAciton[RemitAciton["CONFIRM"] = 1] = "CONFIRM";
        /** 重新提交 */
        RemitAciton[RemitAciton["RESUBMIT"] = 2] = "RESUBMIT";
        /** 关闭 */
        RemitAciton[RemitAciton["CLOSE"] = 3] = "CLOSE";
    })(RemitAciton = Pay.RemitAciton || (Pay.RemitAciton = {}));
    /** 商品ID */
    let ProductIds;
    (function (ProductIds) {
        /** 空间扩容 */
        ProductIds["STORAGE_EXTRA"] = "500001";
        /** 席位扩容 */
        ProductIds["SEAT_EXTRA"] = "500002";
    })(ProductIds = Pay.ProductIds || (Pay.ProductIds = {}));
    /** 商品类型 */
    let ProductType;
    (function (ProductType) {
        /** 主套餐 */
        ProductType[ProductType["PACKAGE"] = 1] = "PACKAGE";
        /** 增购 */
        ProductType[ProductType["EXPANSION"] = 2] = "EXPANSION";
    })(ProductType = Pay.ProductType || (Pay.ProductType = {}));
})(Pay || (Pay = {}));
