// @mt/types-kayn/dist/task.js
/** 异步任务 */
export var Task;
(function (Task) {
    /** 任务类型 */
    let JobType;
    (function (JobType) {
        /** 导出批注 -- word文件 */
        JobType[JobType["Word"] = 0] = "Word";
        /** 导出批注 -- excel文件 */
        JobType[JobType["Excel"] = 1] = "Excel";
        /** 导出文件夹目录 */
        JobType[JobType["DownloadName"] = 3] = "DownloadName";
        /** 代理文件 */
        JobType[JobType["ProxyFile"] = 4] = "ProxyFile";
        /** 水印文件 */
        JobType[JobType["WatterFile"] = 5] = "WatterFile";
        /** 发送至XMP */
        JobType[JobType["SendToXMP"] = 7] = "SendToXMP";
    })(JobType = Task.JobType || (Task.JobType = {}));
    /** 任务状态 */
    let State;
    (function (State) {
        /** 创建 */
        State[State["Create"] = 0] = "Create";
        /** 初始化 */
        State[State["Init"] = 1] = "Init";
        /** 处理中 */
        State[State["Handle"] = 2] = "Handle";
        /** 成功 */
        State[State["Success"] = 3] = "Success";
        /** 错误 */
        State[State["Error"] = 4] = "Error";
    })(State = Task.State || (Task.State = {}));
    /** 任务操作类型 */
    let Action;
    (function (Action) {
        /** 下载 */
        Action[Action["Download"] = 0] = "Download";
        /** 跳转 */
        Action[Action["Jump"] = 1] = "Jump";
    })(Action = Task.Action || (Task.Action = {}));
})(Task || (Task = {}));
