// @mt/types-kayn/dist/userCenter.js
export var UserCenter;
(function (UserCenter) {
    let Ext;
    (function (Ext) {
        let SignAction;
        (function (SignAction) {
            SignAction[SignAction["Register"] = 0] = "Register";
            SignAction[SignAction["Login"] = 1] = "Login";
        })(SignAction = Ext.SignAction || (Ext.SignAction = {}));
        /** web端 新手引导类型，目前针对web端个人版有六个 */
        let SonaGuideType;
        (function (SonaGuideType) {
            SonaGuideType["Global"] = "global";
            SonaGuideType["Project"] = "project";
            SonaGuideType["Resource"] = "resource";
            SonaGuideType["Share"] = "share";
            SonaGuideType["Comment"] = "comment";
            SonaGuideType["Review"] = "review";
            SonaGuideType["AddressBook"] = "address_book";
            SonaGuideType["Interaction"] = "interaction";
            SonaGuideType["CeoLetter"] = "ceo_letter";
            SonaGuideType["Welcome"] = "welcome";
            SonaGuideType["TrialNotification"] = "trial_notification";
            SonaGuideType["AssetViewFocus"] = "asset_view_focus";
            SonaGuideType["AssetViewUploadFocus"] = "asset_view_upload_focus";
            SonaGuideType["AssetViewCommentFocus"] = "asset_view_comment_focus";
        })(SonaGuideType = Ext.SonaGuideType || (Ext.SonaGuideType = {}));
        /** 客户端 新手引导类型*/
        let EkkoGuideType;
        (function (EkkoGuideType) {
            EkkoGuideType["CeoLetter"] = "ceo_letter";
        })(EkkoGuideType = Ext.EkkoGuideType || (Ext.EkkoGuideType = {}));
        /** 小程序端 新手引导类型*/
        let NeekoGuideType;
        (function (NeekoGuideType) {
            NeekoGuideType["Welcome"] = "welcome";
            NeekoGuideType["Home"] = "home";
            NeekoGuideType["Review"] = "review";
            NeekoGuideType["Video"] = "video";
            NeekoGuideType["CeoLetter"] = "ceo_letter";
        })(NeekoGuideType = Ext.NeekoGuideType || (Ext.NeekoGuideType = {}));
    })(Ext = UserCenter.Ext || (UserCenter.Ext = {}));
})(UserCenter || (UserCenter = {}));
